src/client/pages/RetroCertsCertificationPage/index.js

```diff
@@ -195,7 +195,7 @@ function RetroCertsCertificationPage(props) {
   return (
     <div id="overflow-wrapper">
       <Header />
-      <main id="certification-page" className="pb-5 questions">
```

Comment:
can't delete the questions class, if you search for ".questions" across all files you'll see that there's a style applied to it.

Reply:
so you should make this `className="questions"`

Reply:
whoops! sorry, i'll add that back in.